inc/migrate-terms-data.php

```diff
@@ -0,0 +1,288 @@
+<?php
+/**
+ * Data for terms migration
+ */
+
+/**
+ * Get data for transforming WP terms to Hugo terms
+ *
+ * @param int|string $term_id WordPress term ID, integer or numeric string
```

Comment:
You could just leave this as `int`. Even when using strict parameter types in PHP7, specifying an int and providing a numeric string casts the numeric string as an integer.